packages/slice-machine/server/src/api/screenshots/screenshots.ts

```diff
@@ -59,30 +59,15 @@ export default async function handler({
       href
     );
 
-    // We display an error if no screenshot has been taken
-    if (!screenshot) {
-      const message = `Could not generate screenshot for variation ${variationId}`;
-      return {
-        err: new Error(message),
-        reason: message,
-        warning: null,
-        screenshot,
-      };
-    }
-
     return {
-      err: null,
-      reason: null,
       screenshot,
     };
-  } catch (e) {
-    const crashMessage =
-      "Could not generate screenshots for this slice, upload manually a screenshot";
+  } catch (error) {
+    console.error(error);
```

Comment:
Not sure if we want to add any info here so the user knows what the error is related to. E.g. `[screenshots] - error`